### ISSUE TITLE
Create Sign in and Sign up Modal

### ISSUE THREAD

**Issue by Anurag5086** [CodeTrophs/Frontend#334]
Check out the UI Design Below.
**Sign In**
![image](https://user-images.githubusercontent.com/54763478/115112250-b3c66800-9f95-11eb-986b-90d04f3016a1.png)

**Sign Up**
![image](https://user-images.githubusercontent.com/54763478/115112256-bcb73980-9f95-11eb-9f56-dd3c5a7470a0.png)

**Comment by SalmaZakaria** [comment]
Please assign this to me
I am GSSOC'21 Participant

**Comment by Anurag5086** [comment]
@SalmaZakaria Sure, go ahead!

**Comment by Siddhant-K-code** [comment]
currently, "**Sign in with GitHub**" Not Working?

**Comment by Anurag5086** [comment]
> currently, "**Sign in with GitHub**" Not Working?

Run backend as well

**Comment by Siddhant-K-code** [comment]
I was talking about `https://codetrophs.io/` . Do I have to run "Backend" for the deployed website also?

**Comment by Anurag5086** [comment]
> I was talking about `https://codetrophs.io/` . Do I have to run "Backend" for the deployed website also?

Kindly check again. Its working alright.

**Comment by srrathi** [comment]
@Anurag5086 Please consider assigning this issue to me. I've made some progress in it and just have to change some css according to you design.
![image](https://user-images.githubusercontent.com/57089237/116630817-9a4bf580-a971-11eb-9de0-9df6517e3771.png)

**Comment by Siddhant-K-code** [comment]
This issue is already assigned to @SalmaZakaria, surely you can work on it after the update!

**Comment by srrathi** [comment]
Yes you are right @Siddhant-K-code  but I thought as she has not responded from past few days therefore it will be assigned to someone else but it's allright. will wait till the PA and mentor reassign this issue

**Comment by SalmaZakaria** [comment]
@srrathi  I am working on it
Sorry for the late response

**Comment by Anurag5086** [comment]
@SalmaZakaria Cool, just add a screenshot of your progress.

**Comment by Anurag5086** [comment]
@srrathi Looks good, but let's wait for @SalmaZakaria

**Comment by SalmaZakaria** [comment]
@srrathi I will unassign myself and look for another issue to let you be assigned to this one as you have achieved more than me in it
Good Luck

**Comment by Anurag5086** [comment]
@srrathi You can go ahead with this after the other 2 PR's of yours are merged.

**Comment by Anurag5086** [comment]
> @srrathi I will unassign myself and look for another issue to let you be assigned to this one as you have achieved more than me in it
> Good Luck

@SalmaZakaria Its alright. There is nothing to worry. You can take on with this issue if you want.

**Comment by srrathi** [comment]
> > @srrathi I will unassign myself and look for another issue to let you be assigned to this one as you have achieved more than me in it
> > Good Luck
> 
> @SalmaZakaria Its alright. There is nothing to worry. You can take on with this issue if you want.

yes @SalmaZakaria you can.

**Comment by srrathi** [comment]
@Anurag5086 Can I work on this issue now?

**Comment by aryasoni98** [comment]
@srrathi Yes sure ...

**Comment by srrathi** [comment]
> @srrathi Yes sure ...

I want to ask, do I have to do the  frontend part only and leave the gitlab and google button as it is and link them later on when we will add backend to login with them or I've to integrate login with them functionality.

**Comment by srrathi** [comment]
@aryasoni98 Can you please tell me on which page we have to show this modal. So that I can add it on it.

**Comment by aryasoni98** [comment]
> > @srrathi Yes sure ...
> 
> I want to ask, do I have to do the frontend part only and leave the gitlab and google button as it is and link them later on when we will add backend to login with them or I've to integrate login with them functionality.

You have to integrate login with them functionality .

**Comment by aryasoni98** [comment]
> @aryasoni98 Can you please tell me on which page we have to show this modal. So that I can add it on it.

Landing Page and also remove or rename GitHub login to get start button

**Comment by srrathi** [comment]
> > > @srrathi Yes sure ...
> > 
> > 
> > I want to ask, do I have to do the frontend part only and leave the gitlab and google button as it is and link them later on when we will add backend to login with them or I've to integrate login with them functionality.
> 
> You have to integrate login with them functionality .

I'll integrate the github function for login but the one with gitlab and google that involve backend work.How will i do that?

**Comment by subhangi2731** [comment]
@Anurag5086 can I work on this please

**Comment by aryasoni98** [comment]
@subhangi2731 Yes sure
